backend/src/main/dto/TaskDTOs.ts

```diff
@@ -5,6 +5,7 @@ export interface TaskCreationDTO {
   deadlineDate: Date;
   createdDate: Date;
   modifiedDate: Date;
+  //To update in database: make projectId optional
```

Comment:
remove comment and create an issue about it

Reply:
https://github.com/kevin-ghannoum/soen490/issues/112